tests/rest/test_job.py

```diff
@@ -8,19 +9,23 @@
 
 import pytest
 import requests
+from charset_normalizer import from_bytes
```

Comment:
where does this come from? That looks like a new dependency

as far as I understand you are not using `from_bytes`, except overwriting it lower